src/github/repository.ts

```diff
@@ -30,26 +37,37 @@ export const createRepositoryWebhookHandler = async (context: WebhookContext, gi
 			repoUpdatedAt: new Date(repository.updated_at)
 		});
 
-		await subscription.update({ totalNumberOfRepos: (subscription.totalNumberOfRepos || 0) + 1 });
+		await updateRepoCount(subscription);
+		await findOrStartSync(subscription, context.log, false, "partial", subscription.backfillSince,undefined, repository.id);
```

Comment:
`subscription.backfillSince,undefined` <-- there should be a space after the comma?